keyboards/crkbd/keymaps/dvorak_42_key/keymap.c

```diff
@@ -0,0 +1,287 @@
+
```

Comment:
All source files need a GPL2+ license header:
```
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
done, thank you.